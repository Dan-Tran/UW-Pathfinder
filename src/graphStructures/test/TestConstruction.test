# This will test creating the graph and making sure its state is correct.

# Create the graph.
CreateGraph graphtest

# Make sure the graph has no nodes and no edges.
ListNodes graphtest