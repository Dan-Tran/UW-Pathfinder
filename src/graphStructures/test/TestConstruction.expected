# This will test creating the graph and making sure its state is correct.

# Create the graph.
created graph graphtest

# Make sure the graph has no nodes and no edges.
graphtest contains: